README.md

```diff
@@ -1,10 +1,13 @@
-# Solargators Telemetry GUI
-
-On the race track we have data being read and recorded on the solar car. Previously this data was read locally and wasn't moved off the vehicle until after the competition. Now we would like to display the realtime telemetry information on a GUI at the the pit. The telemetry information consists of voltage, current, temperature, GPS data, etc.
+# Solar Gators Telemetry GUI
 
 Please refer to the  [Pit-GUI documentation on readthedocs](https://docs.ufsolargators.org/en/latest/Electrical/Telemetry/gui.html).
 
 
-## GUI Preview
+## Live Status Updates
 
 ![GUI Preview](https://raw.githubusercontent.com/Solar-Gators/docs/master/_static/telemetry/live-telemetry.png)
+
+
+## Easy Strategy Calculations
+
+![pitguidemo](https://github.com/Solar-Gators/Pit-GUI/assets/26682594/ad41b1bc-2300-4b93-9d04-30443bfe481c)
```

Comment:
My only concern is we could be leaking data here, but looks like we're briefly showing RPM, Orion SOC, and our SOC which is fine